Source/SwiftyDropboxUnitTests/GlobalErrorResponseHandlerTests.swift

```diff
@@ -0,0 +1,63 @@
+import XCTest
+@testable import SwiftyDropbox
+
+class GlobalErrorResponseHandlerTests: XCTestCase {
```

Comment:
We should add a test specifically for the auth error even if it may be repetitive

Reply:
Changed it to be an auth error